hack/install-tools.sh

```diff
@@ -9,16 +9,4 @@ source "${OS_ROOT}/hack/common.sh"
 GO_VERSION=($(go version))
 echo "Detected go version: $(go version)"
 
-go get golang.org/x/tools/cmd/cover github.com/tools/godep golang.org/x/tools/cmd/vet
```

Comment:
My bad. We need godep but cover should be gone as well. See https://groups.google.com/forum/#!topic/golang-announce/JMs4_CGbXWk
If you want to fix this now feel free to carry over this PR. I won't be able to update it till later tonight or tomorrow.